### ISSUE TITLE
not able to drag and drop in Android 10.

### ISSUE THREAD

**Issue by Niki01Rud** [gitim/react-native-sortable-list#234]
used "react-native-sortable-list": "^0.0.24", it's working fine in IOS and android except android 10 OS. not able to drag list item.

Attaching my code for referance.

/**
 * Sample React Native App
 * httpss://github.com/facebook/react-native
 * @flow
 */

import React, { Component } from 'react';
import {
  Animated,
  Easing,
  StyleSheet,
  Text,
  Image,
  View,
  Dimensions,
  Platform,
  TouchableOpacity
} from 'react-native';
import SortableList from 'react-native-sortable-list';
import Swipeable from 'react-native-gesture-handler/Swipeable';

const window = Dimensions.get('window');

const data = {
  0: {
    text: '734058957879',
  },
  1: {
    text: '676766545467',
  },
 2: {
    text: '7643666745467',
  },
  
 

};

export default class Basic extends Component {
  render() {
    return (
      <View style={styles.container}>
        <SortableList
          style={styles.list}
          contentContainerStyle={styles.contentContainer}
          data={data}
          renderRow={this._renderRow} />
      </View>
    );
  }

  _renderRow = ({ data, active }) => {
    console.log("_renderRow navigation :", this.props.navigation)
    return <Row navigation={this.props.navigation} data={data} active={active} />
  }
}

class Row extends Component {

  constructor(props) {
    super(props);

    const { navigation } = props.navigation;

    this._active = new Animated.Value(0);

    this._style = {
      ...Platform.select({
        ios: {
          transform: [{
            scale: this._active.interpolate({
              inputRange: [0, 1],
              outputRange: [1, 1.1],
            }),
          }],
          shadowRadius: this._active.interpolate({
            inputRange: [0, 1],
            outputRange: [2, 10],
          }),
        },

        android: {
          transform: [{
            scale: this._active.interpolate({
              inputRange: [0, 1],
              outputRange: [1, 1.07],
            }),
          }],
          elevation: this._active.interpolate({
            inputRange: [0, 1],
            outputRange: [2, 6],
          }),
        },
      })
    };
  }

  componentWillReceiveProps(nextProps) {
    if (this.props.active !== nextProps.active) {
      Animated.timing(this._active, {
        duration: 300,
        easing: Easing.bounce,
        toValue: Number(nextProps.active),
      }).start();
    }
  }

  render() {
    const { data, active } = this.props;

    const leftSwipe = (progress, dragx, navigation) => {
      const scale = dragx.interpolate({
        inputRange: [0, 100],
        outputRange: [0, 1],
        //extrapolateRight:'clamp'
      })
      return (
        <TouchableOpacity onPress={() => {
          navigation.navigate("Home")
        }
        }>

          <View
            style={styles.viewButton}>
            <Animated.Text style={{ color: 'white' }}>View Policy</Animated.Text>
          </View>
        </TouchableOpacity>
      )
    }

    return (
      <Swipeable renderRightActions={(progress, dragx) => {
        console.log("navigation :", this.props.navigation)
        return leftSwipe(progress, dragx, this.props.navigation)
      }}>
        <Animated.View style={[
          styles.row,
          this._style,
        ]}>
          <Text style={styles.text}>Policy Number : </Text>

          <Text style={styles.text}>{data.text}</Text>

        </Animated.View>
      </Swipeable>

    );
  }
}

const styles = StyleSheet.create({
  viewButton: {
    width: 100,
    backgroundColor: 'red',
    justifyContent: 'center',
    alignItems: 'center',
    height: 80,
    marginTop: 7,
    marginRight: 10,
    borderTopRightRadius: 4,
    borderBottomRightRadius: 4
  },
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#eee',

    ...Platform.select({
      ios: {
        paddingTop: 10,
      },
    }),
  },

  title: {
    fontSize: 20,
    color: '#999999',
  },

  list: {
    flex: 1,
  },

  contentContainer: {
    width: window.width,

    ...Platform.select({
      ios: {
        paddingHorizontal: 10,
      },

      android: {
        paddingHorizontal: 0,
      }
    })
  },

  row: {
    flexDirection: 'column',
    backgroundColor: '#fff',
    paddingHorizontal: 10,
    height: 80,
    flex: 1,
    marginTop: 10,
    marginTop: 7,
    borderTopRightRadius: 4,
    borderBottomRightRadius: 4,

    ...Platform.select({
      ios: {
        width: window.width - 10 * 2,
        shadowColor: 'rgba(0,0,0,0.2)',
        shadowOpacity: 1,
        shadowOffset: { height: 2, width: 2 },
        shadowRadius: 2,
      },

      android: {
        width: window.width - 10 * 2,
        elevation: 0,
        marginHorizontal: 10,
      },
    })
  },

  image: {
    width: 50,
    height: 50,
    marginRight: 30,
    borderRadius: 25,
    marginLeft: 20
  },

  text: {
    fontSize: 16,
    color: '#222222',
    margin: 5
  },
});

please help me to resolve this issue, thanks in advance😊

**Comment by MorganTrudeau** [comment]
Yes I see basic example from this repo is not working in android 10.

**Comment by PahmCuogn** [comment]
@Niki01Rud you can apply solution at [https://github.com/gitim/react-native-sortable-list/issues/188#issuecomment-629758136](here), it working on my app
